specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/examples/SuccessfulSentimentRequest.json

```diff
@@ -0,0 +1,326 @@
+                    "text": "atmosphere",
+                    "relations": [
+                      {
+                        "relationType": "opinion",
+                        "links": "#/documents/0/sentences/0/opinions/0"
+                      }
+                    ]
+                  }
+                ],
+                "opinions": [
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 1,
+                      "negative": 0
+                    },
+                    "offset": 0,
+                    "length": 5,
+                    "text": "great",
+                    "isNegated": false
+                  }
+                ]
+              },
+              {
+                "text": "Close to plenty of restaurants, hotels, and transit!",
+                "sentiment": "neutral",
+                "confidenceScores": {
+                  "positive": 0.13,
+                  "neutral": 0.86,
+                  "negative": 0.01
+                },
+                "offset": 18,
+                "length": 52,
+                "aspects": [
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 0.99,
+                      "negative": 0.01
+                    },
+                    "offset": 37,
+                    "length": 11,
+                    "text": "restaurants",
+                    "relations": [
+                      {
+                        "relationType": "opinion",
+                        "links": "#/documents/0/sentences/1/opinions/0"
+                      }
+                    ]
+                  },
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 0.99,
+                      "negative": 0.01
+                    },
+                    "offset": 50,
+                    "length": 6,
+                    "text": "hotels",
+                    "relations": [
+                      {
+                        "relationType": "opinion",
+                        "links": "#/documents/0/sentences/1/opinions/0"
+                      }
+                    ]
+                  }
+                ],
+                "opinions": [
+                  {
+                    "sentiment": "positive",
+                    "confidenceScores": {
+                      "positive": 0.99,
+                      "negative": 0.01
+                    },
+                    "offset": 18,
+                    "length": 15,
+                    "text": "close to plenty",
+                    "isNegated": false
+                  }
+                ]
+              },
+              {
+                "text": "Staff are friendly and helpful.",
+                "sentiment": "positive",
+                "confidenceScores": {
+                  "positive": 1.0,
```

Comment:
Why do we have `1.0` here and `1` at the aspects scores below?

Reply:
Changed to 1.0 and 0.0 for aspect as well.